test/functional/system/common.js

```diff
@@ -157,6 +157,7 @@ function getTransactionFromModule(library, filter, cb) {
 }
 
 function beforeBlock(type, cb) {
+	// eslint-disable-next-line mocha/no-top-level-hooks
```

Comment:
Might make more sense to disable it before the function/at the start of the function and re-enable it after/at the end? Or maybe that's easier to abuse.